dss-docker/Dockerfile

```diff
@@ -59,7 +59,6 @@ RUN R --slave --no-restore \
 
 # Entry point
 WORKDIR /home/dataiku
-USER dataiku
```

Comment:
Please try to avoid as much as possible  execution of run.sh as root 
In the`DockerFile` permissions are already enforced  for the data directory  
If it's a cloud provider level permission  issue  we should consider that enforcing container level related permissions must fix the problem.